x-pack/filebeat/module/cisco/amp/test/cisco_amp.ndjson.log-expected.json

```diff
@@ -0,0 +1,80 @@
+[
+    {
+        "@timestamp": "2020-11-11T09:51:38.000Z",
+        "cisco.amp.computer.active": true,
+        "cisco.amp.computer.connector_guid": "1235-1234sdgf-654sdf-7562345",
+        "cisco.amp.computer.external_ip": "8.8.8.8",
+        "cisco.amp.computer.network_addresses": [
+            {
+                "ip": "192.168.196.22",
+                "mac": "aa:d9:ac:af:1d:ad"
+            },
+            {
+                "ip": "192.168.120.1",
+                "mac": "12:24:56:c2:00:01"
+            },
+            {
+                "ip": "192.168.160.1",
+                "mac": "12:50:56:c2:53:08"
+            }
+        ],
+        "cisco.amp.connector_guid": "1235-1234sdgf-654sdf-7562345",
+        "cisco.amp.detection": "W32.Trojan.22gp.1201",
+        "cisco.amp.detection_id": "12365423467",
+        "cisco.amp.event_type_id": 1090519054,
+        "cisco.amp.file.disposition": "Malicious",
+        "cisco.amp.file.parent.disposition": "Unknown",
+        "cisco.amp.group_guids": [
+            "6542345gdfs-234-sdf2-34-6345243"
+        ],
+        "cisco.amp.related.mac": [
+            "aa:d9:ac:af:1d:ad",
+            "12:24:56:c2:00:01",
+            "12:50:56:c2:53:08"
+        ],
+        "cisco.amp.timestamp_nanoseconds": 153000000,
+        "event.action": "Threat Detected",
+        "event.dataset": "cisco.amp",
+        "event.id": 123578990,
+        "event.kind": "alert",
+        "event.module": "cisco",
+        "event.severity": 2,
+        "file.hash.md5": "9a8557b98ed1469272fa0ace91d63477",
+        "file.hash.sha1": "d0c4192b65e36553fvfd2b83f3113f6ae8390baa",
+        "file.hash.sha256": "e678899d7ea9702184067b56655f91b69f8a0bdc9df65613762252c055c2cdvc",
+        "file.name": "HYXiN3hY.exe.part",
+        "file.path": "\\\\?\\C:\\Users\\elastic\\AppData\\Local\\Temp\\HYXiN3hY.exe.part",
+        "fileset.name": "amp",
+        "host.hostname": "testhost",
```

Comment:
is there any way of filling in the os family? Say by looking at the file path to determine this is windows?

Reply:
I am able to detect windows, but Linux or OSX is going to be more difficult to differentiate.